glide.yaml

```diff
@@ -24,15 +24,15 @@ import:
 
 # We use libcompose to parse v1 and v2 of Docker Compose
 - package: github.com/docker/libcompose
-  version: 4a647d664afbe05c41455c9d534d8239671eb46a
+  version: 57bd716502dcbe1799f026148016022b0f3b989c
```

Comment:
don't update libcompose, you don't need to.

Reply:
Sirupsen :(